packages/medusa-plugin-contentful/src/services/contentful.js

```diff
@@ -142,165 +137,165 @@ class ContentfulService extends BaseService {
   }
 
   async createProductInContentful(product) {
```

Comment:
Handle product creation with collection in Medusa

Reply:
https://vuuzletvph.com/heads-up-vuco-bounty-programs-are-coming-your-way/